frontend/src/metabase/dashboard/selectors.js

```diff
@@ -102,6 +102,20 @@ export const getDashboardComplete = createSelector(
     },
 );
 
+// Auto-apply filters
+export const getDraftParameterValues = state =>
+  state.dashboard.draftParameterValues;
+export const getIsAutoApplyFilters = createSelector(
+  [getDashboard],
```

Comment:
using `getDashboard` the same selector that's used in `frontend/src/metabase/dashboard/actions/parameters.js`